cypress/support/index.js

```diff
@@ -41,3 +41,35 @@ require("cypress-xpath"); // Refer - https://www.npmjs.com/package/cypress-xpath
 // load and register the grep feature
 // https://github.com/bahmutov/cypress-grep
 require("cypress-grep")();
+
+beforeEach(() => {
+    // This wait makes sure that every promise is completed before calling cy.window
+    cy.wait(2000);
+    cy.window().then((win) => {
+        // window.gc is enabled with --js-flags=--expose-gc chrome flag or ELECTRON_EXTRA_LAUNCH_ARGS=--js-flags=--expose_gc
+        if (typeof win.gc === "function") {
+            // run gc multiple times in an attempt to force a major GC between tests
+            win.gc();
+            win.gc();
+            win.gc();
+            win.gc();
+            win.gc();
```

Comment:
I tried that because I saw this issue https://github.com/cypress-io/cypress/issues/8525

After adding that block I observed less tests failing due to Chromium crashing.

Reply:
Finally removed.